docker-compose.yml

```diff
@@ -0,0 +1,79 @@
+version: '3'
+services:
+
+  flask:
+    build:
+      context: app
+      dockerfile: Dockerfile
+    container_name: flask
+    image: flask-python:3.6
```

Comment:
python 3.6 has been deprecated, switch to a supported version, we're now using 3.8+ in other services

https://endoflife.date/python